src/main/java/org/apache/bcel/classfile/ConstantUtf8.java

```diff
@@ -62,6 +62,7 @@ private static class Cache {
 
         private static final boolean BCEL_STATISTICS = Boolean.getBoolean(SYS_PROP_STATISTICS);
         private static final int MAX_ENTRIES = Integer.getInteger(SYS_PROP_CACHE_MAX_ENTRIES, 0).intValue();
+        @SuppressWarnings("cast.unsafe")
```

Comment:
Can you prefix this by a checker name, so that readers know what error is being suppressed?